content/projects/gsoc/2021/project-ideas/plugin-installation-manager-tool.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: gsocprojectidea
+title: "Plugin Installation Manager Tool Improvements"
+goal: "Update the plugin installation manager to integrate with the Configuration as Code plugin, Docker images, and other Jenkins projects"
+category: Tools
+year: 2020
+status: draft
+sig: platform
+skills:
+- Java
+- JSON
+- Command line tools
+- Package management tool theory
+mentors:
+# TBC
+#- "kwhetstone"
+#- "timja"
+- "oleg_nenashev"
+links:
+  gitter: "jenkinsci/plugin-installation-manager-cli-tool"
+  draft: https://docs.google.com/document/d/1s-dLUfU1OK-88bCj-GKaNuFfJQlQNLTWtacKkVMVmHc
+---
+=== Background
+Last summer, there was a Jenkins Google Summer of Code project to create a tool to automatically install plugins from the command line based on a provided input.
+This is meant to help with creating reproducible environments and other places where you would want to install a certain set of plugins.
+The project was successful in creating a useable tool to install plugins with a 1.0 release.
+As the project progressed over the summer, the student received a lot of enhancement suggestions that were great, but out of the scope of the summer project.
+This summer, the project should incorporate enough enhancements to be used by the official Jenkins docker tooling and Configuration as Code.
+
+Following the project tracker, there are several tickets specific to better running with the docker image that would make it more universal.
+Most of these revolve around using alternative locations for downloading plugins, and sharing a cache with the maven repo.
+
+Along with improvements that are already known, it’s important for the student to work with the community to integrate the tool with the docker image and with Configuration as Code.
```

Comment:
the tool is in the docker image probably needs some tweaks throughout the page

Reply:
Yes. Hence a draft